README.md

```diff
@@ -170,6 +170,36 @@ The yaml file contains a custom key of `:document_types` not used by wraith but
 bundle exec rake wraith:update_document_types[:sample_size]
 ```
 
+## Webchat
+
+### How to add a new webchat provider
+1. Open to `lib/webchat.yaml`
+2. Append new entry:
+```yaml
+- base_path: /government/contact/my-amazing-service
+  open_url: https://www.my-amazing-webchat.com/007/open-chat
+  availability_url: https://www.my-amazing-webchat.com/007/check-availability
+```
+3. Deploy changes
+4. Go to https://www.gov.uk/government/contact/my-amazing-service
+5. Finished
+
+### Required APIs / Links
+
+#### Availability API
```

Comment:
Nitpick, I think this heading should be `availability_url` so it matches the key in the yaml.
Same for "Open Link" below, I think that heading should be `open_url`.